Project/Core/Configuration/AppConfig.cs

```diff
@@ -13,6 +13,10 @@ public static class AppConfig
         /// Determines if need to close application on fatal error
         /// </summary>
         public static readonly bool DO_CLOSE_APP_ON_FATAL_ERROR = false;
+        /// <summary>
+        /// Determines if need to delete test database
+        /// </summary>
+        public static readonly bool DO_DELETE_TEST_DB = true;

```

Comment:
currently(in active testing period) we can have **false** here

Reply:
Okay